examples/basic/bars/nested_colormapped.py

```diff
@@ -19,7 +20,7 @@
         '2016'   : [5, 3, 3, 2, 4, 6],
         '2017'   : [3, 2, 4, 4, 5, 3]}
 
-palette = ["#c9d9d3", "#718dbf", "#e84d60"]
+palette = MediumContrast3
```

Comment:
Instead of this assignment, you can just pass `palette=MediumContrast3` directly to `p.vbar(...)` below. That will let us shorten the script by a few lines.

Reply:
Yeah that makes sense!Updating it in a few